RecoLocalFastTime/FTLClusterizer/plugins/MTDClusterProducer.cc

```diff
@@ -109,6 +109,7 @@ MTDClusterProducer::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<double>("HitThreshold", 0.);
   desc.add<double>("SeedThreshold", 0.);
   desc.add<double>("ClusterThreshold", 0.);
+  desc.add<double>("TimeThreshold", 10.);
```

Comment:
This seems like a leftover/incomplete code review from #25063 

 the 4 lines above should be replaced by a single call
```MTDThresholdClusterizer::fillDescriptions(desc);```